runtime/near-vm-errors/src/lib.rs

```diff
@@ -11,6 +11,7 @@ pub enum VMError {
     /// An error that is caused by an operation on an inconsistent state.
     /// E.g. an integer overflow by using a value from the given context.
     InconsistentStateError(InconsistentStateError),
+    CacheError(String),
```

Comment:
It's beneficial to have an enum on strictly typed `CacheError`, instead of relying on free form strings

Reply:
Agree